docs/protocol.html

```diff
@@ -185,17 +185,16 @@ <h5><a id="protocol_common" href="#protocol_common">Common Request and Response
 
 <pre>
 RequestOrResponse => Size (RequestMessage | ResponseMessage)
-Size => int32
+  Size => int32
 </pre>
 
 <table class="data-table"><tbody>
 <tr><th>Field</th><th>Description</th></tr>
 <tr><td>message_size</td><td>The message_size field gives the size of the subsequent request or response message in bytes. The client can read requests by first reading this 4 byte size as an integer N, and then reading and parsing the subsequent N bytes of the request.</td></tr>
 </table>
 
-<h5><a id="protocol_message_sets" href="#protocol_message_sets">Message Sets</a></h5>
-
-<p>A description of the message set format can be found <a href="https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-Messagesets">here</a>. (KAFKA-3368)</p>
+<h5><a id="protocol_recordbatch" href="#protocol_recordbatch">Record Batch</a></h5>
+<p>A description of the record batch format can be found <a href="documentation/#recordbatch">here</a>.</p>
```

Comment:
I am not sure this href will work, but I don't know how to test.

Reply:
We're all over the place with our hrefs, so I can understand the confusion. I think it would probably need to be `/documentation/implementation/#recordbatch`.